AmplifyPlugins/Auth/AWSCognitoAuthPluginIntegrationTests/AuthUserTests/AuthUserTests.swift

```diff
@@ -57,4 +57,175 @@ class AuthUserTests: AWSAuthBaseTest {
         let user = Amplify.Auth.getCurrentUser()
         XCTAssertNotNil(user, "In signedIn state the user should be nil")
     }
+
+    /// Test attribute fetch in signedout state
```

Comment:
Nit: consider moving the "signed out" state tests to a separate test suite file

Reply:
I have another pending changes with signOut related tests - https://github.com/aws-amplify/amplify-ios/tree/royjit/SessionIntegTest .